src/services/web-monitoring-db.js

```diff
@@ -207,6 +207,17 @@ export default class WebMonitoringDb {
   }
 
   /**
+   * Get a single version
+   * @param {string} versionId
+   * @returns {Promise<Version>}
+   */
+  getVersionById (versionId) {
+    return this._request(this._createUrl(`versions/${versionId}`))
+      .then(response => response.json())
+      .then(data => parseVersion(data.data));
+  }
+
+  /**
```

Comment:
I don’t think you should add this method. You should remove the `pageId` parameter from `getVersion` (it predates the ability to actually get a version without a page ID).

(Sorry again for missing so many things before. I should not have done a midnight review.)